doc/changes.rst

```diff
@@ -51,6 +51,8 @@ Varnish Cache 7.x.x (2021-09-15)
 
 * ACLs default to `pedantic` which is now a per-ACL feature flag.
 
+* `varnishncsa -j` won't refuse to print fields with control characters anymore.
+
```

Comment:
I really hate this kind of double negatives in documentation...

Reply:
I don't see nothing wrong with that, but I'll change that and merge